opencog/spacetime/octomap/octomap.scm

```diff
@@ -0,0 +1,8 @@
+(define-module (opencog octomap))
+
+(use-modules (srfi srfi-1) (opencog) (opencog exec))
+
+; Load the C library that calls the nameserver to load the types.
+(load-extension "libatom_octomap" "opencog_spacetime_octomap_init")
```

Comment:
and you don't need this call to `opencog_spacetime_octomap_init` (because it doesn't do anything)